### PR TITLE
test publishing workflow

### PR DESCRIPTION
**Pull request by fahdarafat**

### CODE REVIEW

File: embed/embedding.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
+/* eslint-disable */
+(function () {
+  init();
+  createIframe();
+})();
+function init() {
+  sb.p = sb.p || {};
+  for (p of sb.s) {
+    sb.css = "../../assets/embed.css";
```
Comment (Ibrahimogod) [inline embed/embedding.js:9]: embedding script is gonna be stand alone in githubpage are we sure this is gonna work

File: frontend/src/views/NotificationsIndex.vue [hunk 2]
```diff
@@ -21,10 +21,11 @@ const notifications = ref([])
 const archivedNotifications = ref([])
 const subscriberID = ref('')
 
-const socket = io('http://127.0.0.1:3000', {
+const socket = io(process.env.VUE_APP_SERVER_BASE_URL, {
```
Comment (Ibrahimogod) [inline frontend/src/views/NotificationsIndex.vue:24]: is this gonna make conflicts with the other branch we've created

Comment (fahdarafat) [reply frontend/src/views/NotificationsIndex.vue:24]: Yess, But I will resolve it there

File: embed/embedding.js (hunk 1 above)
Comment (fahdarafat) [reply embed/embedding.js]: I removed it. I was copied from the original embedding script and I don't think we need it here

**Review by Ibrahimogod — APPROVED** [review]

File: embed/embedding.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+(function () {
+  init();
+  createIframe();
+})();
+function init() {
+  sb.p = sb.p || {};
+  for (p of sb.s) {
+    if (p[0] === "url") sb.url = p[1];
+    if (p[0] === "subscriberId") sb.subid = p[1];
+    if (p[0] === "realmHeader") sb.realm = p[1];
```
Comment (MoeFarag) [inline embed/embedding.js:11]: why do you call it `realmHeader`,, why not just `realm`

File: embed/embedding.js [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    if (p[0] === "theme") sb.t = p[1];
+    if (p[0] === "bellOptions") {
```
Comment (MoeFarag) [inline embed/embedding.js:13]: sara7a i hate the specifity of it being `bell`,, just use `icon` all over

**Review by MoeFarag — CHANGES_REQUESTED** [review]

**Review by MoeFarag — APPROVED** [review]

**Pull request merged**